tests/test_utils/gcp_system_helpers.py

```diff
@@ -68,3 +77,92 @@ def provide_gcp_context(
     return provide_gcp_conn_and_credentials(
         key_file_path=key_file_path, scopes=scopes, project_id=project_id
     )
+
+
+class GoogleSystemTest(SystemTest):
+    @staticmethod
```

Comment:
```suggestion
    @property
```
It's a simple getter. so it should be property.